frontend/src/components/common/Filter/components/FilterContent/components/FilterMenu/index.tsx

```diff
@@ -66,46 +58,45 @@ export default function FilterMenu({
   }, []);
 
   return (
-    <MenuWrapper menuWidth={menuWidth} ref={menuRef}>
-      <Menu>
-        {/* Optional search bar */}
-        {isSearchable && (
-          <FilterSearch
-            searchValue={searchValue}
-            setSearchValue={setSearchValue}
-          />
-        )}
+    <Menu
+      isMenuDivided={isMenuDivided}
+      menuWidth={menuWidth}
+      ref={menuRef}
+      scrollable={scrollable}
+    >
+      {/* Optional search bar */}
+      {isSearchable && (
+        <FilterSearch
+          searchValue={searchValue}
+          setSearchValue={setSearchValue}
+        />
+      )}
+      <List>
         {/* No matches */}
         {emptyItems ? (
-          <NoMatches
-            multiline /* required when no matches text length is longer than longest category value */
-            shouldDismissPopover={false}
-            text={"No matches found"}
-          />
+          <NoMatches>No matches found</NoMatches>
         ) : (
-          <MenuItemsScroller {...scrollerProps}>
+          <>
             {/* Pinned values */}
             {filteredPinnedValues.length > 0 && (
               <FilterMenuItems
                 categoryFilterId={categoryFilterId}
-                isMultiselect={isMultiselect}
                 menuItems={filteredPinnedValues}
                 onFilter={onFilter}
               />
             )}
             {/* Menu divider */}
-            {isMenuDivided && <MenuDivider />}
+            {isMenuDivided && <Divider />}
```

Comment:
Reverting here and elsewhere to either core SDS or Mui components and targeting the styling of these components via a ancestor.

This is firstly to improve rendering performances, but also to create a single source of truth for filtering styles that can be shared amongst the various menu components.